### PR TITLE
Console chat cosmetic changes. Fixes #281, #284, #285, #287

### LINKED ISSUE #281: Onboarding Flow & Console Cosmetic Changes
- [x] "console" should have uppercase Console
- [x] Console status should be "Chat with me to setup your account and change your settings!"
- [x] Console is missing DApp indicator icon (blue double dots, please generalise to ensure that DApp contacts always have this)

screen shot 2016-09-29 at 1 26 05 pm

- [x] Console icon needs to change (find attached, let me know if I have provide different assets)

console
console 0 5x
console 0 25x

- [x] First 3 messages simplified to "Welcome to Status\nTap this message to set your password & get started!"

- [x] Messages after !password should look as follows
screen shot 2016-09-29 at 2 10 08 pm
- Here is your passphrase, **write this down and keep this safe!** You will need it to recover your account.
- 12 word passphrase
- Tap here to enter your phone number & I'll find your friends

### LINKED ISSUE #284: Chat input field icons and text not aligned
screen shot 2016-09-29 at 1 38 53 pm

Top image is from emulator, bottom is from design

Notice how the command icon is smaller and inline, i imagine the text needs to change, please ensure all are vertically aligned in middle.

### LINKED ISSUE #285: Tapping command list icon in onboarding = Red Screen
Steps to reproduce
- Run Application
- Tap Add Account
- Tap Command list Icon

= 
screen shot 2016-09-29 at 1 40 37 pm

found on develop

### LINKED ISSUE #287: Command Messages missing !,  Password using **
screen shot 2016-09-29 at 2 06 17 pm

- [x] keypair should be !password
- [x] password using *** instead of ●●●
- [x] font looks off? is it Roboto Regular?

### PR DESCRIPTION
**Pull request by rasom**
fix #281 
fix #284 
fix #285 
fix #287

### CODE REVIEW

File: src/status_im/chat/styles/message_input.cljs [hunk 1]
```diff
@@ -4,27 +4,26 @@
...
-   :flex             1
-   :flex-direction   "column"
-   :margin-right     0})
+(defn message-input-container []
+  {:height         16.5
+   :margin-top     20
+   :margin-bottom  19.5
+   :flex           1
+   :flex-direction :column
+   :margin-right   0})
```
Comment (alwx) [inline src/status_im/chat/styles/message_input.cljs]: not important or obligatory, but you can replace it with `(def message-input-container { ... })`

File: src/status_im/chat/styles/message_input.cljs [hunk 2]
```diff
@@ -4,27 +4,26 @@
...
-(defn input-view [content-height]
+(defn input-view []
   {:flex-direction   :row
    :align-items      :center
    :justify-content  :center
-   :height           (+ (min (max min-input-height content-height) max-input-height)
-                        input-spacing-top
-                        input-spacing-bottom)
+   :height           56
    :background-color color-white})
```
Comment (alwx) [inline src/status_im/chat/styles/message_input.cljs]: not important or obligatory, but you can replace it with `(def input-view { ... })`

File: src/status_im/models/chats.cljs [hunk 3]
```diff
@@ -88,6 +76,7 @@
 (defn create-chat
   ([{:keys [last-message-id] :as chat}]
    (let [chat (assoc chat :last-message-id (or last-message-id ""))]
+     (println :UIUIOUOI chat)
```
Comment (alwx) [inline src/status_im/models/chats.cljs]: hm :)

Comment (rasom) [inline src/status_im/models/chats.cljs]: oops, thanks

Comment (rasom) [inline src/status_im/models/chats.cljs]: fixed

**Pull request merged**